### PR TITLE
Redesign bug fixes

### PR DESCRIPTION
**Pull request by stacywcmc**
Multiple fixes for the redesign. Can you keep the branch open once merged?

### CODE REVIEW

File: app/assets/stylesheets/utilities/typography.scss [hunk 1]
```diff
@@ -24,6 +43,11 @@ p.lead {
   margin: auto;
 }
 
-ul li {
-  padding: 0.5em 0;
+ul {
```
Comment (lucidstack) [inline app/assets/stylesheets/utilities/typography.scss]: Uhm what is this stuff doing in a `typography.scss` file? Can we find a better place?

Comment (stacywcmc) [inline app/assets/stylesheets/utilities/typography.scss]: Yep, apologies. It was a bit of a reflex. I am used to having base styles for HTML elements in the same scss file. Will move.

Comment (stacywcmc) [inline app/assets/stylesheets/utilities/typography.scss]: @lucidstack moved!

Comment (lucidstack) [inline app/assets/stylesheets/utilities/typography.scss]: Nice! 🎉

**Pull request merged**